docs/messaging/azure-service-bus-component.md

```diff
@@ -180,12 +185,47 @@ The .NET Aspire Azure Service Bus component uses the following log categories:
 
 - `Azure.Core`
 - `Azure.Identity`
+- `Azure-Messaging-ServiceBus`
 
 ### Tracing
 
-The .NET Aspire Azure Service Bus component will emit the following tracing activities using OpenTelemetry:
-
-- "Azure.Data.Tables.TableServiceClient"
+> [!NOTE]
+> Service Bus `ActivitySource` support in the Azure SDK for .NET is experimental, and the shape of activities may change in the future without notice.
+
+You can enable tracing in one of three ways:
+
+- Set the `DisableTracing` option to `false` in the configuration.
+- Call `System.AppContext.SetSwitch("Azure.Experimental.EnableActivitySource", true)`.
```

Comment:
````suggestion
- Setting the `Azure.Experimental.EnableActivitySource` [runtime configuration setting](https://learn.microsoft.com/dotnet/core/runtime-config/) to `true`. Which can be done with either:
    - Call `AppContext.SetSwitch("Azure.Experimental.EnableActivitySource", true);`.
    - Add the RuntimeHostConfigurationOption setting to your project file:
```xml
<ItemGroup>
    <RuntimeHostConfigurationOption Include="Azure.Experimental.EnableActivitySource" Value="true" />
</ItemGroup>
```
````

Reply:
See https://devblogs.microsoft.com/azure-sdk/introducing-experimental-opentelemetry-support-in-the-azure-sdk-for-net/#get-started

Reply:
FWIW https://github.com/davidfowl/Build2024AspireDemo/blob/main/AppHost/ServiceBusTracingExtensions.cs